### DISCUSSION TITLE
Why `useLockFile: false` for `npm install` in GitHub Actions?

### DISCUSSION THREAD

**Discussion by crswll** [epicweb-dev/epic-stack#23] [General]
I noticed the jobs in GitHub Actions use `useLockFile: false` for `npm install` and I'm curious about the motivation. I thought using that would mean quicker and more consistent builds. Is it used as a way to detect issues earlier on? Is it not necessary any longer?

**Comment by kentcdodds** [comment]
Thanks for bringing this up. That was an old copy/paste error I never noticed 🙈 I've removed that now.
